tokens/src/components/Carousel.json

```diff
@@ -54,23 +55,17 @@
         "bg": {
           "prev-icon": {
             "value": "url(\"data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='{color.carousel.control.base.value}' viewBox='0 0 8 8'%3e%3cpath d='M5.25 0l-4 4 4 4 1.5-1.5-2.5-2.5 2.5-2.5-1.5-1.5z'/%3e%3c/svg%3e\")",
+            "type": "file",
```

Comment:
Hmm, I don't think we should use `file` here, we're not loading any files with the `url`, it's just an svg. Maybe we can introduce `svg` type for such cases, even though it's not covered by W3C specs?

@adamstankiewicz curious what you think

Reply:
@viktorrusakov Good question... I'm a bit hesitant to introduce a `svg` type that isn't officially covered by the W3C spec. The file type per the W3C spec seems to indicate the `file` type might still be in flux as well. Given SVGs are technically files, even though its not defined as a separate file here, I'm inclined to leave these tokens as `"type": "file"` for the time being.